test/utils/fixtures/index.ts

```diff
@@ -722,7 +725,7 @@ export const seaportFixture = async (owner: Wallet) => {
         if (consideration.itemType === 1) {
           // ERC20
           // search for transfer
-          const transferLogs = (tokenEvents || [])
+          const transferLogs = (tokenEvents ?? [])
```

Comment:
Minor: I think you should share with contributors the difference between `||` and `??` as ppl seemed to often use `||` throughout the test files in places where `??` was likely intended (if they knew about it) for "using right side only if left side is null or undefined"... and I have a feeling ppl will keep doing this is if it's not brought up haha

Reply:
Hehe yeah the nullish coalescing operator is really great. Where would you suggest I add this communication? I could add this lint rule actually to help enforce: https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/prefer-nullish-coalescing.md